README.md

```diff
@@ -44,278 +44,316 @@ Unrecommended ones can still be considered. The most important thing is to remai
 
 ### A
 
--  🟢 abbr • abbreviation
--  🟢 abs • absolute number
--  🔴 act • action / active / actual
--  🟢 addr • address
--  🟡 alloc • allocate {memory}
--  🟢 alt • alternative / alternate
--  🟢 app • application
--  🟡 arg • argument {memory}
--  🟢 arr • array
--  🟢 async • asynchronous
--  🟢 attr • attribute
--  🟢 auth • authentication / authenticate
--  🟢 avg • average
+-  abbreviation • 🟢 abbr
+-  absolute • 🟢 abs
+-  addition • 🟢 sum
+-  address • 🟢 addr
+-  action • 🔴 act
+-  active • 🔴 act
+-  actual • 🔴 act
+-  allocate • 🟡 alloc {memory}
+-  alternative • 🟢 alt
+-  alternate • 🟢 alt
+-  application • 🟢 app
+-  array • 🟢 arr
+-  argument • 🟢 arg
+-  asynchronous • 🟢 async
+-  attribute • 🟢 attr
+-  authentication • auth
+-  average • 🟢 avg
 
 ### B
 
--  🟡 bat • battery {power}
--  🟢 bg • background
--  🟢 bin • binary
--  🟡 bl • breakline {encoding}
--  🟢 bool • boolean
--  🔴 brk • break
--  🟢 btn • button
--  🟢 buf • buffer
+-  background • 🟢 bg
+-  battery • 🟡 bat {power}
+-  binary • 🟢 bin
+-  boolean • 🟢 bool
+-  break • 🔴 brk
+-  breakline • 🟡 bl {encoding}
+-  buffer • 🟢 buf
+-  button • 🟢 btn
 
 ### C
 
--  🟢 calc • calculate / calculator
--  🟢 cb • callback
--  🟢 cert • certificate
--  🟡 ch • channel {connection}
--  🟢 chk • check
--  🟢 char • character
--  🟡 circ • circle {shape}
--  🟢 clr • clear
--  🟢 cmd • command
--  🟡 cmp • compare {conditional operator}
--  🟢 cnt • counter
--  🟢 concat • concatenate
--  🟢 conf / config • configuration
--  🟢 conn • connection {connection}
--  🟢 cont • continue
--  🟢 conv • conversation
--  🟢 col • column
--  🟢 coll • collection
--  🔴 com • commercial / common / communication
--  🟢 coord • coordinate
--  🟢 cos • cosine
--  🟢 cot • cotangent
--  🟡 csum • checksum {operation}
--  🟢 ctrl • control
--  🟢 ctx • context
--  🟢 cur / curr • current
--  🔴 cpy • copy
+-  calculate • 🟢 calc
+-  calculator • 🟢 calc
+-  callback • 🟢 cb
+-  certificate • 🟢 cert
+-  certification • 🟢 cert
+-  channel • 🟡 ch {connection}
+-  character • 🟢 char
+-  check • 🟢 chk
+-  checksum • 🟡 csum {operation}
+-  circle • 🟡 circ {shape}
+-  clear • 🟢 clr
+-  command • 🟢 cmd
+-  commercial • 🔴 com
+-  common • 🔴 com
+-  communication • 🔴 com
+-  compare • 🟡 cmp {conditional operator}
+-  collection • 🟢 coll
+-  column • 🟢 col
+-  concatenate • 🟢 concat
+-  configuration
+   -  🟢 conf
+   -  🟢 config
+-  connection • 🟢 conn {connection}
+-  context • 🟢 ctx
+-  continue • 🟢 cont
+-  control • 🟢 ctrl
+-  conversation • 🟢 conv
+-  coordinate • 🟢 coord
+-  copy • 🔴 cpy
+-  cosine • 🟢 cos
+-  cotangent • 🟢 cot
+-  counter • 🟢 cnt
+-  current
+   -  🟢 cur
+   -  🟢 curr
```

Comment:
Same here, but I don't know which one is better.

Reply:
As they don't different much from each other I suggest "curr"